### PR TITLE
Add capybara to application

### PR DESCRIPTION
**Pull request by roman-dubrovsky**
#100

### CODE REVIEW

File: spec/features/main_page_spec.rb [hunk 1]
```diff
@@ -0,0 +1,144 @@
+require "rails_helper"
+
+describe "main page integration test" do
+  before do
+    visit("/")
+  end
+
+  after(:all) do
```
Comment (roman-dubrovsky) [inline spec/features/main_page_spec.rb]: I din't have any idea how to fix it so I'm going to add issue for this after merge

Comment (AleksSenkou) [inline spec/features/main_page_spec.rb]: @roman-dubrovsky can't database cleaner help us here?

File: spec/features/main_page_spec.rb [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+      it "shows final step description" do
+        within_active_step do
+          check_steps_description(description)
+          check_steps_button("Say Hello!", :green)
+        end
+      end
+    end
+  end
+end
```
Comment (AleksSenkou) [inline spec/features/main_page_spec.rb]: @roman-dubrovsky very clear tests ❤️

File: spec/features/main_page_spec.rb (hunk 1 above)
Comment (roman-dubrovsky) [inline spec/features/main_page_spec.rb]: @AleksSenkou no, it couldn't

**Pull request merged**